src/player-interaction.js

```diff
@@ -195,6 +199,10 @@ class PlayerInteraction {
       name = 'raise';
       amount = input[1] ? parseInt(input[1]) : NaN;
       break;
+    case 'a':
```

Comment:
'a' is a little too easy to type by accident. Since this is a nuclear option, let's only look for the strings "all in," "all-in," and possibly "allin."